hugolib/author.go

```diff
@@ -13,23 +13,30 @@
 
 package hugolib
 
+import (
+	"strings"
+
+	"github.com/spf13/cast"
+)
+
 // AuthorList is a list of all authors and their metadata.
 type AuthorList map[string]Author
 
 // Author contains details about the author of a page.
 type Author struct {
-	GivenName   string
-	FamilyName  string
-	DisplayName string
-	Thumbnail   string
-	Image       string
-	ShortBio    string
-	LongBio     string
-	Email       string
-	Social      AuthorSocial
+	GivenName   string            // given_name
```

Comment:
Not sure what the comments are meant to be here, but if you suggest some kind of "XML or JSON" naming here, you might as well add JSON tags here so it gets used when people do `{{ author | jsonify }}`